pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -1445,7 +1445,7 @@ self: {
        license = stdenv.lib.licenses.bsd3;
        hydraPlatforms = stdenv.lib.platforms.none;
        broken = true;
-     }) {inherit (pkgs) db; inherit (pkgs) dbxml; 
```

Comment:
This file is generated automatically and would have to be fixed in https://github.com/NixOS/cabal2nix

Reply:
that's fine if it gets overwritten, this is just to avoid the situation where CI will block on minor linting issues

Reply:
This file is ignored by the editorconfig check.
https://github.com/NixOS/nixpkgs/blob/3ffc05e1dba2874cbb30b1ebc0e3dfe531e06249/.editorconfig#L91-L93